examples/controls_example/public/react_controls/data_controls/initialize_data_control.ts

```diff
@@ -49,6 +53,10 @@ export const initializeDataControl = <EditorState extends object = {}>(
   const fieldName = new BehaviorSubject<string>(state.fieldName);
   const dataViews = new BehaviorSubject<DataView[] | undefined>(undefined);
   const filters$ = new BehaviorSubject<Filter[] | undefined>(undefined);
+  const field$ = new BehaviorSubject<DataViewField | undefined>(undefined);
+  const fieldFormatter = new BehaviorSubject<DataControlFieldFormatter>((toFormat: any) =>
+    String(toFormat)
+  );
```

Comment:
**All** data controls should need access to their field + corresponding formatter, so it made sense to move this to the generic data control.